test/tools/verify/cmd/main.go

```diff
@@ -27,10 +30,17 @@ func main() {
 	var manifestPath string
 	var parameters string
 	var kubeConfig string
+	var createOnly bool
 
-	flag.StringVar(&kubeConfig, "kubeconfig", "", "a path to the kubeconfig.")
+	defaultKubeconfigPath, err := getDefaultKubeconfigPath()
+	if err != nil {
+		log.Fatalf("Error getting default path for kubeconfig: '%v'", err)
```

Comment:
insteaf of using log, please use glog, as its the library used by the kubernetes go client